specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -512,12 +935,12 @@
           "description": "Rounded value."
         },
         "unit": {
+          "x-ms-client-name": "unitLabel",
           "type": "string",
           "description": "Type of unit for the returned value."
         },
         "unitType": {
-          "type": "integer",
-          "format": "int32",
+          "$ref": "#/definitions/UnitType",
           "description": "Numeric ID value associated with the type of unit being displayed. Can be used for unit translation. Please refer to [Weather Service Concepts](https://aka.ms/AzureMapsWeatherConcepts) for details."
```

Comment:
This `description` can be deleted since it is also in the `UnitType` definition.